guide/guide.html

```diff
@@ -789,11 +789,11 @@ <h3>Methods</h3>
                     <pre>
 Dog := Object clone
 Dog bark := method("woof!" print)
-</pre> The above code creates a new "subclass" of object named Dog and adds a bark slot containing a block that prints "woof!". Example of calling this method:
+</pre> The above code creates a new "subclass" of object named Dog and adds a bark slot containing a method that prints "woof!". Example of calling this method:
 
                     <pre>
 Dog bark
-</pre> The default return value of a block is the result of the last expression.
+</pre> The default return value of a method is the result of the last expression.
```

Comment:
Since this is in the **Methods** section, and that section is followed by a separate **Blocks** section, it seemed more proper to be consistent in referring to these as "methods".

That said, maybe this is a squares/rectangles thing internally and all methods are blocks but not all blocks are methods or something of that nature. If that's the case, though, I still think consistency within these sections is better.